src/main/java/io/numaproj/numaflow/mapper/Message.java

```diff
@@ -2,57 +2,53 @@
 
 import lombok.Getter;
 
-/**
- * Message is used to wrap the data returned by Mapper.
- */
-
+/** Message is used to wrap the data returned by Mapper. */
 @Getter
-public class Message {
-    public static final String DROP = "U+005C__DROP__";
-
-    private final String[] keys;
-    private final byte[] value;
-    private final String[] tags;
-
-
-    /**
-     * used to create Message with value, keys and tags(used for conditional forwarding)
-     *
-     * @param value message value
-     * @param keys message keys
-     * @param tags message tags which will be used for conditional forwarding
-     */
-    public Message(byte[] value, String[] keys, String[] tags) {
-        this.keys = keys;
-        this.value = value;
-        this.tags = tags;
-    }
-
-    /**
-     * used to create Message with value.
-     *
-     * @param value message value
-     */
-    public Message(byte[] value) {
-        this(value, null, null);
-    }
-
-    /**
-     * used to create Message with value and keys.
-     *
-     * @param value message value
-     * @param keys message keys
-     */
-    public Message(byte[] value, String[] keys) {
-        this(value, keys, null);
-    }
-
-    /**
-     * creates a Message which will be dropped
-     *
-     * @return returns the Message which will be dropped
-     */
-    public static Message toDrop() {
-        return new Message(new byte[0], null, new String[]{DROP});
-    }
+public final class Message {
+  private static final String[] DROP_TAGS = {"U+005C__DROP__"};
```

Comment:
just curious, why do we use this notation for DROP_TAGS?

Reply:
Good question Krithika, first it's to match what we defined [on platform side](https://github.com/numaproj/numaflow/blob/3e54bb5e26dd8a7aef28a1be546f85b8dfb8cf6c/pkg/apis/numaflow/v1alpha1/const.go#L304) as platform use it to drop messages. Regarding why the exact notation was chosen, I don't know, I feel it's just a const representing drop, which is unlikely to be specified by users.